roles/docker/templates/docker-svc.j2

```diff
@@ -12,7 +12,7 @@ ExecStart=/usr/bin/docker daemon --storage-opt dm.thinpooldev=/dev/mapper/contiv
 {% else %}
 ExecStart=/usr/bin/docker daemon -H fd:// --cluster-store=etcd://localhost:{{ etcd_client_port1 }}
 {% endif %}
-MountFlags=slave
+MountFlags=shared
```

Comment:
@vvb 

Yes, both volplugin and k8s need this mode. I didn't really see a good usecase for `slave` during my search so I left it hardcoded. Let me make it a variable.